Move Fate Points section to new chapter (#73)
MiniFate/MiniFate: _chapters/05-aspects-and-fate-points.md

### Fate Points

Fate Points are a currency that can be spent to assert narrative control over

> the game. Both the players and the GM have pools of Fate Points available to
> spend. Players start each session with a number of Fate Points equal to their

Reviewer: ```suggestion
the game. Each player has a pool of fate points to spend from, as does the
GM. Players start each session with a number of Fate Points equal to their
```

PC's Refresh; additional Fate Points are earned by accepting compels during
the session.

> The GM start each scene with as many Fate Points as there are PCs active in

Reviewer: ```suggestion
The GM starts each scene with one fate point for each PC in
```
